recipe_scrapers/kitchensanctuary.py

```diff
@@ -0,0 +1,78 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers._grouping_utils import IngredientGroup
+
+from ._abstract import AbstractScraper
+
+
+class KitchenSanctuary(AbstractScraper):
```

Comment:
This site supports `cook_time` and `prep_time` in the schema - can you please add the follow functions to leverage this:
```py
def cook_time(self):
    return self.schema.cook_time()

def prep_time(self):
    return self.schema.prep_time()
```